src/config/SPRACINGH7EXTREME/config.h

```diff
@@ -60,6 +60,7 @@
 
 #define FLASH_QUADSPI_INSTANCE    QUADSPI
 
+#define USE_SDCARD_SDIO
```

Comment:
Is this already defined: https://github.com/betaflight/betaflight/blob/master/src/main/target/common_pre.h#L263-L265 ?